tests/rptest/clients/rpk.py

```diff
@@ -224,8 +224,11 @@ def create_topic():
                 self._check_stdout_success(output)
                 return (True, output)
             except RpkException as e:
+                self._redpanda.logger.debug(str(e))
                 if "Kafka replied that the controller broker is -1" in str(e):
                     return False
+                elif "SASL authentication failed" in str(e):
+                    return False
```

Comment:
LGTM, but I wonder if a more generic fix inside `self._run_topic` might save time in the future.